tools/schema_registry_tools/schemas_command_generator.py

```diff
@@ -60,21 +59,31 @@ def generate_options():
     j = json.loads(tmp)
 
     # Generating rpk topic create commands
+    expected_offset = 0
     cmd_all = ''
     cmd_all += f"#!/bin/bash\n\n"
     for c, i in enumerate(j):
```

Comment:
I think the loop would be simpler if it iterated from 0 to `j[len(j)-1]['offset']`.

It might be easier to follow along if the variable names had more meaning.

Reply:
I have left the `expected_offset` variable as is since I need to track it independently from the json object (`j`). Let me know if this is ok. 
Btw, I've also made the variable names more meaningful in eb7447b.